src/python/pants/backend/python/target_types.py

```diff
@@ -504,6 +504,15 @@ class PexLayoutField(StringField):
     )
 
 
+class PexIncludeRequirementsField(BoolField):
+    alias = "include_requirements"
+    default = True
+    help = (
+        "Whether to include the third party requirements the binary depends on in the "
+        "packaged PEX file."
```

Comment:
Maybe mention a sentence for why this can be useful?

> this can be useful, for example, if you provide the dependencies through the external environment and want a smaller binary.